src/renault_api/kamereon/__init__.py

```diff
@@ -311,17 +320,28 @@ async def set_vehicle_action(
     vin: str,
     endpoint: str,
     attributes: Dict[str, Any],
+    actions: str = "actions",
```

Comment:
If you add an argument in the middle of the list, it becomes a breaking change.
You have to add it at the bottom of the list to make it non-breaking (even with a default value).

I suggest:
```python
    attributes: Dict[str, Any],
    endpoint_version: Optional[int] = None,
    data_type: Optional[Dict[str, Any]] = None,
    *,
    car_adapter_type: str = "kca"
    actions: str = "actions",
```